src/UnityExtension/Assets/Editor/GitHub.Unity/UI/BranchesView.cs

```diff
@@ -536,95 +348,81 @@ private void OnTreeNodeGUI(BranchTreeNode node)
                         }
                     }
-                    else if (node.Type == NodeType.RemoteBranch)
-                    {
-                        var indexOfFirstSlash = selectedNode.Name.IndexOf('/');
-                        var originName = selectedNode.Name.Substring(0, indexOfFirstSlash);
-                        var branchName = selectedNode.Name.Substring(indexOfFirstSlash + 1);
+                });
 
-                        if (localBranches.Any(localBranch => localBranch.Name == branchName))
-                        {
-                            EditorUtility.DisplayDialog(WarningCheckoutBranchExistsTitle, 
-                                String.Format(WarningCheckoutBranchExistsMessage, branchName),
-                                WarningCheckoutBranchExistsOK);
-                        }
-                        else
-                        {
-                            var confirmCheckout = EditorUtility.DisplayDialog(ConfirmCheckoutBranchTitle, 
-                                String.Format(ConfirmCheckoutBranchMessage, node.Name, originName), 
-                                ConfirmCheckoutBranchOK, ConfirmCheckoutBranchCancel);
-
-                            if (confirmCheckout)
-                            {
-                                GitClient.CreateBranch(branchName, selectedNode.Name)
-                                    .FinallyInUI((success, e) =>
-                                    {
-                                        if (success)
-                                        {
-                                            Redraw();
-                                        }
-                                        else
-                                        {
-                                            EditorUtility.DisplayDialog(Localization.SwitchBranchTitle,
-                                                String.Format(Localization.SwitchBranchFailedDescription, node.Name),
-                                                Localization.Ok);
-                                        }
-                                    }).Start();
-                            }
-                        }
-                    }
-                }
+            if (treeHadFocus && treeRemotes.SelectedNode == null)
+            {
+                treeLocals.Focus();
             }
+            else if (!treeHadFocus && treeRemotes.SelectedNode != null)
+            {
+                treeLocals.Blur();
+            }
+
+            if (treeRemotes.RequiresRepaint)
+                Redraw();
+
+            //Debug.LogFormat("reserving: {0} {1} {2}", rect.y - initialRect.y, rect.y, initialRect.y);
+            GUILayout.Space(rect.y - initialRect.y);
         }
 
-        private void OnTreeNodeChildrenGUI(BranchTreeNode node)
+        private int CompareBranches(GitBranch a, GitBranch b)
         {
-            if (node == null || node.Children == null)
+            //if (IsFavorite(a.Name))
```

Comment:
meant to be commented out? Should it just be removed?

Reply:
That is getting cleaned up in #468